addons/portal_rating/static/src/js/portal_rating_composer.js

```diff
@@ -85,6 +85,12 @@ const RatingPopupComposer = publicWidget.Widget.extend({
         // Instantiate the "Portal Composer" widget and insert it into the modal
         this._composer = new PortalComposer(this, this.options);
         return this._composer.appendTo(this.$('.o_rating_popup_composer_modal .o_portal_chatter_composer')).then(() => {
+            // Remove border-0 and border class when the review msg box is in pop up view
```

Comment:
Yes, that’s better with the border around the textarea and that’s the case in 16.0 indeed. So I think your solution is good. 

The only thing I would change is to not use “name” to target the div, as name are used for form element and anchor not to style element. I would rather use a class. 

And as there is already a class, why not use it as follow ?

```javascript
// Set border around the textarea instead of the portal composer body
const textarea =  this.el.querySelector(".o_portal_chatter_composer_body > div textarea[name=\"message\"]");
if (textarea) {
    textarea.parentElement.classList.remove("border");
    textarea.classList.remove("border-0");
}
```